webapp/channels/src/components/copy_button.tsx

```diff
@@ -4,18 +4,16 @@
 import classNames from 'classnames';
 import React, {useRef, useState} from 'react';
 import {Tooltip} from 'react-bootstrap';
-import {FormattedMessage, useIntl} from 'react-intl';
+import {FormattedMessage, defineMessages, useIntl} from 'react-intl';
 
 import OverlayTrigger from 'components/overlay_trigger';
 
 import Constants from 'utils/constants';
-import {t} from 'utils/i18n';
 import {copyToClipboard} from 'utils/utils';
 
 type Props = {
     content: string;
-    beforeCopyText?: string;
-    afterCopyText?: string;
+    isForText?: boolean;
```

Comment:
This component started out being a button just for copying code from code blocks and then was changed to be used in the preview of text files. If we end up using this in more places, it might make sense to invert this prop